src/loaders/JsonImageInfoLoader.ts

```diff
@@ -67,46 +67,55 @@ type JsonImageInfo = {
 };
 /* eslint-enable @typescript-eslint/naming-convention */
 
-const convertImageInfo = (json: JsonImageInfo): ImageInfo => ({
-  name: json.name,
-
-  originalSize: new Vector3(json.width, json.height, json.tiles),
-  atlasTileDims: new Vector2(json.cols, json.rows),
-  volumeSize: new Vector3(json.tile_width, json.tile_height, json.tiles),
-  subregionSize: new Vector3(json.tile_width, json.tile_height, json.tiles),
-  subregionOffset: new Vector3(0, 0, 0),
-  physicalPixelSize: new Vector3(json.pixel_size_x, json.pixel_size_y, json.pixel_size_z),
-  spatialUnit: json.pixel_size_unit || "μm",
-
-  numChannels: json.channels,
-  channelNames: json.channel_names,
-  channelColors: json.channel_colors,
-
-  times: json.times || 1,
-  timeScale: json.time_scale || 1,
-  timeUnit: json.time_unit || "s",
-
-  numMultiscaleLevels: 1,
-  multiscaleLevel: 0,
-  multiscaleLevelDims: [
-    {
-      shape: [json.times || 1, json.channels, json.tiles, json.tile_height, json.tile_width],
-      spacing: [json.time_scale || 1, 1, json.pixel_size_z, json.pixel_size_y, json.pixel_size_x],
-      spaceUnit: json.pixel_size_unit || "μm",
-      timeUnit: json.time_unit || "s",
-      dataType: "uint8",
-    },
-  ],
+const rescalePixelSize = (json: JsonImageInfo): [number, number, number] => {
+  // the pixel_size_x/y/z are the physical size of the original pixels represented by
+  // width and height.  We need to get a physical pixel size that is consistent
+  // with the tile_width and tile_height.
+  const px = (json.pixel_size_x * json.width) / json.tile_width;
+  const py = (json.pixel_size_y * json.height) / json.tile_height;
+  const pz = json.pixel_size_z;
+  return [px, py, pz];
+};
 
-  transform: {
-    translation: json.transform?.translation
-      ? new Vector3().fromArray(json.transform.translation)
-      : new Vector3(0, 0, 0),
-    rotation: json.transform?.rotation ? new Vector3().fromArray(json.transform.rotation) : new Vector3(0, 0, 0),
-  },
+const convertImageInfo = (json: JsonImageInfo): ImageInfo => {
+  const [px, py, pz] = rescalePixelSize(json);
+  // translation is in pixels that are in the space of json.width, json.height.
+  // We need to convert this to the space of the tile_width and tile_height.
+  const tr: [number, number, number] = json.transform?.translation ? json.transform.translation : [0, 0, 0];
```

Comment:
can use nullish coalescing operator here?

```suggestion
  const tr: [number, number, number] = json.transform?.translation ?? [0, 0, 0];
```